ui/libs/__init__.py

```diff
@@ -35,6 +50,19 @@ class Tokens:
 """
 
 
+@staticmethod
```

Comment:
@staticmethod applies only for methods in class

Reply:
Good caught, removed.